common/wrappers/cross-cc

```diff
@@ -23,13 +23,13 @@ while [ $i -lt ${#ARGS[@]} ]; do
 			MYARGS+=("-L${arg}")
 		fi
 		unset libpath
-	elif [ "$arg" = "-I" ]; then
+	elif [ "$arg" = "-I" -o "$arg" = "-isystem" ]; then
```

Comment:
This changes all `-isystem` to `-I`, which changes the search order that might be used for headers.